OpenRA.Mods.Common/Widgets/HotkeyEntryWidget.cs

```diff
@@ -24,6 +24,7 @@ public class HotkeyEntryWidget : Widget
 		public int LeftMargin = 5;
 		public int RightMargin = 5;
 
+		public Action OnEscKey = () => { };
```

Comment:
A small followup PR will be able to use this to unfocus the lobby chat and fix #15426.